xiaomi.vacuum.gen1/firmwarebuilder/imagebuilder.sh

```diff
@@ -43,12 +42,12 @@ fi
 
 if [[ $# -eq 0 ]]; then
 	cat << EOF
-usage: sudo ./firmwarebuilder -f v11_003094.pkg [-s english.pkg] [-k authorized_keys] [ -t Europe/Berlin ] [--disable-xiaomi]
+usage: sudo ./firmwarebuilder -f v11_003094.pkg [-s english.pkg] [-k id_rsa.pub ] [ -t Europe/Berlin ] [--disable-xiaomi]
```

Comment:
Please add example in which form you can add several public-keys as parameter. I'm not sure if I need to give the absolute or relative path to the file. Also how do you add several keys? comma/space separated?

Reply:
Alright, added!

Reply:
awesome, thank you! I never expected it to be multiple -k parameters so it's a good addition :)